src/main/engine/GfxController/headers/OpenGlGfxController.hpp

```diff
@@ -15,6 +15,8 @@
 #include <GfxController.hpp>
 #include <Polygon.hpp>
 #include <common.hpp>
+// Temporary until we get a logger, disables noisy OpenGL logs
+// #define VERBOSE_LOGS
```

Comment:
This is just temporary until we have a proper logger. I just needed OpenGlGfxController to stay quiet.